.github/workflows/Pipeline.yml

```diff
@@ -12,6 +12,7 @@ jobs:
     uses: pyTooling/Actions/.github/workflows/Parameters.yml@r0
     with:
       name: pyEDAA.UCIS
+      system_list: "ubuntu windows macos"
```

Comment:
Why is testing on msys2 disabled?

Reply:
It's currently an issue with installing pytest and other packages.
I would transform the `Install` job to a job template in pyTooling/Actions and then see how we can use pacboy to solve the problem.

Reply:
As commented in https://github.com/edaa-org/pyEDAA.UCIS/pull/3/files/009895161b986e1d23f3fe06bfd43d107f814c8f..003a69a140e4a074373efc988fa2f85c98acd06e#r786890774, the `Install` job is essentially `UnitTesting`. Therefore, I don't think a new job template is required. It's just a matter of adding an option there. Since the repository is cloned regardless of how the package is installed, it is straightforward to make `install` an option in UnitTesting.

Reply:
Oh, maybe we can merge these usecases.

These are the changes:
1. download the package artifact
2. install the package
3. run from a different directory `tests/console`
4. different artifact name for the test report xml.

Reply:
Is there a reason to run the `tests/unit` and `tests/console` in different jobs? Can't we just run both of them at the same time (and generate a single artifact)?

Reply:
In the UnitTesting job, the package does not exist for installation.  
Installation with `setup.py` is not sufficient.